### PR TITLE
Integrating mlflow

### PR DESCRIPTION
**Pull request by opro-pl**
Added MLFlow integration into the Melkor project; 
added MLFlow local model tracking, model versioning, shared model loading through PyFunc; 
added evaluation metrics

### CODE REVIEW

File: melkor/utils/utils.py [hunk 1]
```diff
@@ -1,7 +1,60 @@
...
+    metrics["rmse"] = mean_squared_error(y_true, y_predicted, squared=False)
+
+    metrics["r2"] = r2_score(y_true, y_predicted)
+    metrics["explained_variance"] = explained_variance_score(y_true, y_predicted)
+    metrics["max_error"] = max_error(y_true, y_predicted)
+    return metrics
+
+
+def eval_classification(
+    y_true: Union[list, np.array, pd.Series],
```
Comment (jmen-pl) [inline melkor/utils/utils.py:35]: Document

File: melkor/utils/utils.py [hunk 2]
```diff
@@ -1,7 +1,60 @@
...
+    f1_score,
+)
+from typing import Union
+import numpy as np
+import pandas as pd
 import yaml
 from collections import defaultdict
 
 
+def eval_regression(
```
Comment (jmen-pl) [inline melkor/utils/utils.py:20]: Document

**Review by jmen-pl — COMMENTED** [review]
As a general remark, please always documentt

**Comment by jmen-pl** [comment]
@tehr-pl @opro-pl this has already been open for 4 days

**Comment by opro-pl** [comment]
Added all the documentation to the new modules.

**Pull request merged**